### ISSUE TITLE
Uncaught TypeError: Cannot read property 'setAttribute' of undefined

### ISSUE THREAD

**Issue by uehondor** [pusher/atom-pair#34]
[Enter steps to reproduce below:]
1. Cmd+Shift+P
2. Type "Start new pairing"
3. Press "Enter" and I get the following error

**Atom Version**: 0.188.0
**System**: Mac OS X 10.10.2
**Thrown From**: [atom-pair](https://github.com/pusher/atom-pair) package, v1.1.2
### Stack Trace

Uncaught TypeError: Cannot read property 'setAttribute' of undefined

```
At .../.atom/packages/atom-pair/lib/atom_pair.coffee:125

TypeError: Cannot read property 'setAttribute' of undefined
    at Object.module.exports.AtomPair.pairingSetup (.../.atom/packages/atom-pair/lib/atom_pair.coffee:181:38)
    at Object.module.exports.AtomPair.startSession (.../.atom/packages/atom-pair/lib/atom_pair.coffee:171:21)
    at atom-workspace.subscriptions.add.atom.commands.add.AtomPair:start new pairing session (.../.atom/packages/atom-pair/lib/atom_pair.coffee:83:26)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/command-registry.js:242:29)
    at /opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/command-registry.js:3:61
    at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/node_modules/command-palette/lib/command-palette-view.js:120:32)
    at CommandPaletteView.module.exports.SelectListView.confirmSelection (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
    at space-pen-div.atom.commands.add.core:confirm (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/command-registry.js:242:29)
    at /opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/command-registry.js:3:61
```
### Commands

```
     -1:16.3 window:update-available (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-monokai.theme-atom-dark-ui)
     -0:52.0 command-palette:toggle (atom-pane.pane.active)
     -0:43.5 core:backspace (atom-text-editor.editor.mini)
     -0:43.4 editor:delete-to-beginning-of-line (atom-text-editor.editor.mini)
     -0:38.8 core:confirm (atom-text-editor.editor.mini)
     -0:38.8 AtomPair:invite over slack (atom-pane.pane.active)
     -0:37.8 core:paste (div)
     -0:27.5 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-monokai.theme-atom-dark-ui)
     -0:04.1 command-palette:toggle (div.settings-view.pane-item)
     -0:01.5 core:backspace (atom-text-editor.editor.mini)
     -0:00.7 core:confirm (atom-text-editor.editor.mini)
     -0:00.7 AtomPair:start new pairing session (div.settings-view.pane-item)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "monokai"
    ],
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store"
    ],
    "disabledPackages": [
      "vim-mode"
    ],
    "excludeVcsIgnoredPaths": false
  }
}
```
### Installed Packages

``` coffee
# User
atom-jshint, v1.5.0
atom-pair, v1.1.2
behat-atom, v0.1.2
file-icons, v1.5.2
monokai, v0.13.0
php-checkstyle, v0.6.1
php-twig, v3.0.0
phpunit, v1.0.9
visual-bell, v1.0.0

# Dev
No dev packages
```

**Comment by jpatel531** [comment]
Ah! Were you on the settings page when you started? Try opening a new tab and starting. In the meantime I'll patch it so that it doesn't throw an error when on the settings page :)

**Comment by uehondor** [comment]
Cheers mate! When you release the fix, I'll check it out.

**Comment by jpatel531** [comment]
no prob! I just released version 1.1.3 which should resolve the issue :)
